src/Components/SystemDetail/RecommendationsTable.js

```diff
@@ -59,7 +60,21 @@ const renderExpandedView = (row) => {
                             {suggestedInstances}</Td>
                     </Tr>
                 </Tbody>
-            </TableComposable> }
+            </TableComposable>
+            }
+            {
+                !psiEnabled &&
+                <>
+                    <hr/>
+                    <Text component={TextVariants.p}>
+                        <Text className="margin-text-bottom"><LightbulbIcon/>
+                            <strong className="strong-tag-style">Related Knowledgebase Article</strong>
+                        </Text>
+                        {/* eslint-disable-next-line max-len */}
+                        <a href={ENABLE_PSI_URL} target='_blank' rel="noreferrer">This suggestion could be improved by enabling PSI <ExternalLinkAltIcon/></a>
```

Comment:
@r14chandra FYI, we can also use Text component here https://www.patternfly.org/v4/components/text/#visited